tests/AccountTest.php

```diff
@@ -34,10 +34,7 @@ function invalid_signups_dont_proceed()
     {
         $this->visit('register')
             ->press('Sign up')
-            ->seePageIs('register')
-            ->see('The name field is required')
-            ->see('The password field is required')
-            ->see('The email field is required');
+            ->seePageIs('register');
```

Comment:
Are we no longer seeing those errors?

Reply:
Ah, yes we do see those errors. I've re-added them and also the errors are now displayed below the text inputs.

![Screenshot 2019-09-04 at 21 15 53](https://user-images.githubusercontent.com/191121/64288232-7dbba800-cf59-11e9-848c-b3e09926aae8.png)